src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -55,6 +56,21 @@ public string AzureFilesContentShare
                 ? Environment[EnvironmentSettingNames.AzureFilesContentShare]
                 : SiteName;
 
+        public string FunctionsWorkerRuntime =>
```

Comment:
Move this to EnvironmentExtensions - so can be reused at other places as well.

Reply:
Done.